tests/test_util.py

```diff
@@ -501,6 +501,16 @@ def test_complex_conditions_expr(self, dataset):
             **locals()
         )
 
+        # Messages can have newlines in them
```

Comment:
Please add an equivalent test for the AST in this file
https://github.com/getsentry/snuba/blob/master/tests/query/parser/test_expressions.py#L11

Reply:
Added a test to https://github.com/getsentry/snuba/blob/master/tests/query/parser/test_conditions.py instead since this isn't an aggregate. Let me know if I missed the mark here